www/src/pages/docs/node-api-helpers.js

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { graphql, Link } from "gatsby"
+import Helmet from "react-helmet"
+import sortBy from "lodash/sortBy"
+
+import APIReference from "../../components/api-reference"
+import { rhythm, scale } from "../../utils/typography"
+import Layout from "../../components/layout"
+import Container from "../../components/container"
+import { itemListDocs } from "../../utils/sidebar/item-list"
+
+class NodeAPIHelperDocs extends React.Component {
+  render() {
+    const docs = sortBy(
+      this.props.data.allDocumentationJs.edges.map(({ node }) => node),
+      docs => docs.name
+    )
+    return (
+      <Layout location={this.props.location} itemList={itemListDocs}>
+        <Container>
+          <Helmet>
+            <title>Node API helpers</title>
+          </Helmet>
+          <h1 id="gatsby-node-helpers-apis" css={{ marginTop: 0 }}>
+            Gatsby Node API helpers
+          </h1>
+          <p>
+            The first argument passed to each of{` `}
+            <Link to="/docs/node-apis/">Gatsby's Node APIs</Link> is an object
```

Comment:
It would help to see a code example here to provide more context. How does the "object containing a set of utilities" relate to this list of helpers? Are the helpers passed into Gatsby Node APIs?

Reply:
Yes, the first argument passed to Gatsby Node APIs is object containing those "helpers", I will try to clarify that.

I also started to wonder if this should be called `context` instead of `helpers` (might be less confusing?).

Reply:
Is it the same context that we see elsewhere? If so, then it would be good to be consistent. Otherwise, the naming is less important than showing it in use.

Reply:
There plenty of different contexts, and we never used `context` to describe first param in gatsby-node APis :( pages have their context, graphql have their context, so I'll just drop this idea